### PR TITLE
Socint 55 dockerise rh UI

### PR DESCRIPTION
**Pull request by PeterBochel**
This change adds scripts and docker-compose files to allow RH UI and its dependencies to run in Docker.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Running RH UI in Docker
+
+Running the RH UI Service (and it's dependencies) in Docker saves any problems with 
+local builds and making sure that you are using the correct Python version.
+
+## Prerequisits
```
Comment (philwhiles) [inline docker/README.md:6]: Prerequisites

File: docker/docker-compose-mock-ai.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+version: '3.8'
+services:
+  mock-ai:
```
Comment (philwhiles) [inline docker/docker-compose-mock-ai.yml:3]: Have commented to Peter that mock-ai is now deprecated and not even built by concourse any more.
New JIRA to be created by Peter for immed action to remedy this with a switch for this docker suite to use mock-service instead

Comment (PeterBochel) [reply docker/docker-compose-mock-ai.yml:3]: SOCINT-123 has been created to update rh-ui and rh-service when mock-case is completed.

File: docker/README.md (hunk 1 above)
Comment (PeterBochel) [reply docker/README.md]: Typo fixed

**Review by rpcatling — APPROVED** [review]

**Review by KieranWardle — APPROVED** [review]

**Pull request merged**